applications/CompressiblePotentialFlowApplication/custom_processes/apply_far_field_process.cpp

```diff
@@ -0,0 +1,129 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Iñigo Lopez and Marc Núñez
```

Comment:
I think we unfortunately need to put Inigo and Nunez without the ñ :(

Reply:
I thought it was OK since it was a comment, but I guess some editors might not accept the ñ. There are more files with ñ's though, we can remove them in future PRs